sdk/schemaregistry/schema-registry/review/schema-registry.api.md

```diff
@@ -0,0 +1,96 @@
+## API Report File for "@azure/schema-registry"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+
+// @public (undocumented)
+export interface SchemaId {
+    // (undocumented)
+    id: string;
+}
+
+// @public (undocumented)
+export class SchemaRegistryClient extends SchemaRegistryClientContext {
+    constructor($host: string, options?: SchemaRegistryClientOptionalParams);
+    getIdBySchemaContent(groupName: string, schemaName: string, serializationType: string, schemaContent: string, options?: coreHttp.OperationOptions): Promise<SchemaRegistryClientGetIdBySchemaContentResponse>;
+    getSchemaById(schemaId: string, options?: coreHttp.OperationOptions): Promise<SchemaRegistryClientGetSchemaByIdResponse>;
+    registerSchema(groupName: string, schemaName: string, serializationType: string, schemaContent: string, options?: coreHttp.OperationOptions): Promise<SchemaRegistryClientRegisterSchemaResponse>;
```

Comment:
is `serializationType` the payload format e.g. `json`/`avro`?

Reply:
I believe so. cc @arerlend

Reply:
Yup there is a bit of flakiness on whether we are referring to this encoding format, e.g. JSON or Avro, as 'serialization type' or 'schema type'.  Would be good to make a choice on Monday at the arch-board review so we can make a final call.